stable/insights/templates/_helpers.tpl

```diff
@@ -25,11 +25,11 @@ If release name contains chart name it will be used as a full name.
 {{- end -}}
 
 {{- define "insights.influxdb_url" -}}
-{{- $context  := dict "Values" .Values.influxdb "Chart" (dict "Name" "influxdb") "Release" .Release  "Capabilities" .Capabilities -}}
-{{- if and .Values.influxdb .Values.influxdb.enabled -}}
+{{- $context  := dict "Values" .Values.influxdb2 "Chart" (dict "Name" "influxdb") "Release" .Release  "Capabilities" .Capabilities -}}
```

Comment:
Is `"Chart" (dict "Name" "influxdb")` still correct as the new Helm chart is `influxdb2`?

Reply:
Thanks for this catch. Solved the nameOverride issue.